src/pca_report_library/compiler/xelatex.py

```diff
@@ -26,10 +26,8 @@
 from typing import Dict
 
 # Third-Party Libraries
-from docopt import docopt
-
-# cisagov Libraries
 from _version import __version__
```

Comment:
This line should move out of the "Third-Party Libraries" section and look like this:
```suggestion
from ._version import __version__
```

For reference, see the [skeleton](https://github.com/cisagov/skeleton-python-library/blob/develop/src/example/example.py#L28-L33).